src/operation/aggregate/mod.rs

```diff
@@ -95,11 +100,18 @@ impl Operation for Aggregate {
             wc_error_info.validate()?;
         };
 
+        let comment = if description.max_wire_version.unwrap_or(0) < SERVER_4_4_0_WIRE_VERSION {
```

Comment:
why do we need this check here and for the `find` operation? it seems like outside of cursors we're just relying on the server to error if comment is unsupported, what makes these cases different?

Reply:
The `AggregateOptions` (specification)[https://github.com/mongodb/specifications/blob/master/source/crud/crud.rst#read] (and also the FindOptions] requires that the comment for an `aggregate` / `find` operation be propagated in subsequent `getMore` operations for server version 4.4+. This was needed to account for that.

Reply:
ahh I see. thanks! I think it could be helpful to add a brief comment in each of these places explaining that on < 4.4 `find` / `aggregate` support comments but `getMore` doesn't so we can't propagate it.